### ISSUE TITLE
min_cpu_platform doesn't work on node pools

### ISSUE THREAD

**Issue by dantheperson** [terraform-google-modules/terraform-google-kubernetes-engine#1216]
**Actual**
All node pools take the min_cpu_platform of the first node pool
**Expected**
Node pools takes the min_cpu_platform defined in their node pool declaration.

https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/modules/private-cluster/cluster.tf#L284

\- min_cpu_platform = lookup(var.node_pools[0], "min_cpu_platform", "")
+min_cpu_platform = lookup(each.value, "min_cpu_platform", "")

Looks like the bug is across all the different cluster modules

For example with this configuration, both node pools have min_cpu_platform="AMD Rome"

```
  node_pools = [
    {
      name               = "platform"
      machine_type       = "n2d-standard-4"
      min_cpu_platform   = "AMD Rome

    },
    {
      name               = "elasticsearch"
      machine_type       = "n2d-standard-4"
      min_cpu_platform   = "AMD Milan"

    },
```

**Comment by lauraseidler** [comment]
I think the fix for this is in #1207

**Comment by dantheperson** [comment]
Indeed, #1207 looks like it'll fix it to me.
